lib/moonchild.js

```diff
@@ -186,6 +195,16 @@ function getEditor() {
   return globalEditor;
 }
 
+function getChannel() {
```

Comment:
I don't think it belongs in the editor, in my live branch for example, the channel is used for other purposes besides loading and saving.